source/js/StartContainer.js

```diff
@@ -4,11 +4,13 @@
  * @author Andy Young
  * @author Justin Lee
  * @author Liam Stone
+ * @author Chris Yoon
  */
 
 import StartButtons from './StartButtons.js';
 import UsernameInput from './UsernameInput.js';
 import WelcomeMessage from './WelcomeMessage.js';
+import { get } from './backend.js';
```

Comment:
For the sake of consistency, do you think we go with importing these functions individually or should we go with how Luke and I have been doing it?